src/components/CardContainer/CardContainer.js

```diff
@@ -30,12 +30,28 @@ class CardContainer extends React.Component {
             .then(res => {
                 let random = UniqueRandomArray(res)
                 let dogs = [random(), random()]
-                const state = {...this.state, dogs: dogs}
+                const state = {...this.state, dogs: dogs, randomDog: random}
```

Comment:
as this is a function with an action, can we call this `getRandomDog` - otherwise it looks like a property and is just a random dog

Reply:
On it

Reply:
On it